src/command.c

```diff
@@ -96,8 +96,8 @@ const command_s cmd_list[] = {
 #endif
 #ifdef ENABLE_RTT
 	{"rtt", cmd_rtt,
-		"[enable|disable|status|channel [0..15 ...]|ident [STR]|cblock|ram [RAM_START RAM_END]|poll [MAXMS MINMS "
-		"MAXERR]]"},
+		"[enable|disable|status|channel [0..15 ...]|ident [STR]|send [STR]|cblock|ram [RAM_START RAM_END]|"
+		"poll [MAXMS MMINMS AXERR]]"},
```

Comment:
```suggestion
		"poll [MAXMS MINMS MAXERR]]"},
```
Is this with the `pre-commit` hook installed and clang-format working?